src/Makefile

```diff
@@ -32,8 +32,8 @@ ifeq ($(OS),Windows_NT)
 	endif
 else ifeq ($(COMP),mingw)
 	target_windows = yes
-	ifeq ($(WINE_PATH),)
-		WINE_PATH = $(shell which wine)
+	ifeq ($(EMU_PATH),)
```

Comment:
What about using `EMULATOR_PATH` explicitly, instead of the abbreviation ? 

However, we also have SDE_PATH, and the E there is actually Emulator as well...

Reply:
Updated.
I fear than a change of `SDE_PATH` will disrupt the Abrok workflow.

Reply:
I don't think abrok uses SDE_PATH, but might use WINE_PATH actually. What I wanted to say is that SDE is also an emulator, but somehow (needs to be?) is treated differently.

Reply:
We introduced WINE_PATH only few weeks ago.

There is the theoretical use case of chaining 2 emulators...
`PGOBENCH = $(L0_EMULATOR_PATH) -- $(L1_EMULATOR_PATH) ./$(EXE) bench`

If we exclude this rare case we can try to have only one rule for a single emulator.

Reply:
probably we can indeed have a single EMULATOR_PATH, special setups should be able to define PGOBENCH explicitly.. I'm using SDE_PATH but can adjust the testing script. I wonder about the double dash `--`, it must have been needed with SDE, but it causes troubles under wine...

Reply:
I'm away from the keyboard now, the double dash usually signals the finish of the command parameters.